From d3a4272d313e07284eea19ebe5257456417da750 Mon Sep 17 00:00:00 2001
From: Rob Johnston <rob.johnston@joyent.com>
Date: Thu, 28 Feb 2019 16:40:04 +0000
Subject: [PATCH] OS-7560 libtopo's XML serialization code is broken and
 incomplete (fix unused var) OS-7609 fmtopo is missing code to handle
 properties of type TOPO_TYPE_FMRI_ARRAY (fix unused var)

---
 usr/src/lib/fm/topo/libtopo/common/topo_xml.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/usr/src/lib/fm/topo/libtopo/common/topo_xml.c b/usr/src/lib/fm/topo/libtopo/common/topo_xml.c
index 6b459642e6..3e3241c8f3 100644
--- a/usr/src/lib/fm/topo/libtopo/common/topo_xml.c
+++ b/usr/src/lib/fm/topo/libtopo/common/topo_xml.c
@@ -228,7 +228,7 @@ xlate_common(topo_mod_t *mp, xmlNodePtr xn, topo_type_t ptype, nvlist_t *nvl,
 	double dbl;
 	uint_t i = 0, nelems = 0;
 	nvlist_t *fmri;
-	xmlChar *str, *estr;
+	xmlChar *str;
 	char **strarrbuf;
 	void *arrbuf;
 	nvlist_t **nvlarrbuf;
-- 
2.21.0

